docs/notebooks/diagnostics/line_integrated_diagnostic.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>It would be very nice to have a repeat of the same thing with a higher resolution detector plate, and an explanation of how the pixelated distribution is the result of the chosen resolution.</p><p> </p><p>By the way, if you end the last line with </p><p><code>cb.ax.set_ylabel("Phase Shift / $\pi$", fontsize=16); #note the semicolon</code></p><p> </p><p>It will suppress the output of <code>Text(0, 0.5, 'Phase Shift / $\\pi$')</code>  as the last object returned from the notebook cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/1123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Thanks for the semicolon tip! 

Unfortunately this is already pretty high resolution: I'll double it but it doesn't look THAT much better. The restriction is the size of the array that we need to evaluate the integrand at: for a [h#, v#] detector plane and num=[n#] integration points that's an [h#, v#, n#] array, so it can become slow if the resolution gets high.

Reply:
In that case we could go the other way - start with a smaller resolution and compare with the current one?